data/json/monsters/zed_soldiers.json

```
@@ -332,7 +332,6 @@
     "burn_into": "mon_zombie_scorched",
     "flags": [
       "SEES",
```

```suggestion
    "flags": [ "SEES", "WARM", "GRABS", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "PUSH_MON", "NOHEAD", "FILTHY" ],
```
did you not lint your changes?